src/Microsoft.DocAsCode.Build.Engine/SingleDocumentBuilder.cs

```diff
@@ -75,12 +76,16 @@ private Manifest BuildCore(DocumentBuildParameters parameters)
                     parameters.Files.DefaultBaseDir);
                 if (ShouldTraceIncrementalInfo)
                 {
+                    string configHash = ComputeConfigHash(parameters);
+                    string templateHash = parameters.TemplateManager.GetTemplatesHash();
```

Comment:
will pass templatemanager as documentbuilder's ctor argument

---
In reply to: [75418336](https://github.com/dotnet/docfx/pull/621#discussion_r75418336) [](ancestors = 75418336,75337615)